spec/furniture/marketplace/marketplaces_controller_request_spec.rb

```diff
@@ -23,13 +23,16 @@
   end
 
   describe "#update" do
-    before { sign_in(space, member) }
+    before do
+      sign_in(space, member)
+      freeze_time
```

Comment:
Does this mean time is frozen for the remainder of the thread? Should we be using `around` instead of `before`?

Reply:
My understanding is that the default behavior is that it automatically calls `travel_back` after the tests, so time gets unfrozen at that point: https://api.rubyonrails.org/v5.2.4/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-after_teardown

Reply:
:the-more-you-know.gif: